README.md

```diff
@@ -945,6 +946,60 @@ information
 `ghw.TopologyNode` struct if you'd like to dig deeper into the NUMA/topology
 subsystem
 
+### Accelerator
+
+The `ghw.Accelerator()` function returns a `ghw.AcceleratorInfo` struct that contains
+information about the host computer's processing accelerator hardware. In this category
+we can find used hardware for AI. The hardware detected in this category will be
+processing accelerators (PCI class `1200`), 3D controllers (`0302`) and Display
+controllers (`0380`).
+
+The `ghw.AcceleratorInfo` struct contains one field:
+
+* `ghw.AcceleratorInfo.Devices` is an array of pointers to `ghw.AcceleratorDevice`
+  structs, one for each processing accelerator card found for the system.
+
+Each `ghw.AcceleratorDevice` struct contains the following fields:
+
+* `ghw.AcceleratorDevice.Address` is the PCI address for the processing accelerator card.
+* `ghw.AcceleratorDevice.PCIDevice` is a pointer to a `ghw.PCIDevice` struct.
+  describing the processing accelerator card. This may be `nil` if no PCI device
+  information could be determined for the card.
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/jaypipes/ghw"
+)
+
+func main() {
+	accel, err := ghw.Accelerator()
```

Comment:
do we want to extend (maybe later?) this API to accept additive user-provided filters?
I'm referring to the wide variety of devices out there and how they present themselves on PCI bus.
By "additive" filter I mean
- users cannot rule out a device `ghw` is really sure it's an accelerator (this means `ghw` must be conservative in the core logic, which I think it's good anyway)
- users can opt-in devices if they know better than `ghw` for whatever reasons (maybe old `ghw` version, maybe rare/custom devices)

Reply:
perfect! I'll be on the lookout this time!